pkg/systemd/logs.js

```diff
@@ -414,13 +420,20 @@ $(function() {
                 id = entry.PROBLEM_BINARY;
             }
 
-            $('#journal-entry-id').text(id);
+            $('#journal-entry-heading').text(id);
+
+            const crumb = $("#journal-entry-crumb");
+            const date = moment(new Date(entry.__REALTIME_TIMESTAMP / 1000));
 
             if (is_problem) {
+                crumb.text(cockpit.format("$0: crash @ $1", id, date.format("YYYY-MM-DD HH:mm:ss")));
```

Comment:
"crash" needs to be i18n'ed. Also, please let's not use "@" but a proper word ("crashed at $1") and let that be part of the translation.